src/main.cc

```diff
@@ -503,7 +503,10 @@ load_cfg(void)
 	 */
 	say_info("%s %s %s", tarantool_package(), tarantool_version(),
 		 BUILD_INFO);
-	say_info("log level %i", cfg_geti("log_level"));
+	if (cfg_isnumber("log_level"))
+		say_info("log level %i", cfg_geti("log_level"));
+	else
+		say_info("log level \"%s\"", cfg_gets("log_level"));
```

Comment:
I guess we'd better put this log in log.lua. Say after `say_logger_init` call. AFAIK currently we handle different types for log options on Lua level. So let's keep it there.

Reply:
Changed to `box_say_log_level()`